cmd/entrypoint/runner_test.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"os"
+	"syscall"
+	"testing"
+)
+
+func TestRealRunnerSignalForwarding(t *testing.T) {
+	rr := realRunner{}
+	rr.signals = make(chan os.Signal, 1)
+	rr.signals <- syscall.SIGINT
+	if err := rr.Run("sleep", "3600"); err.Error() == "signal: interrupt" {
+		t.Logf("SIGINT forwarded to Entrypoint")
+	} else {
+		t.Fatalf("Unexpected error received: %v", err)
+	}
+}
```

Comment:
i think im not familiar enough with this to understand how this test case demonstrates that things are working properly - is it possible to add a bit more comments explaining things?

also double checking, this test case fails without your changes?

Reply:
I will make this more verbose and add more comments. I did miss out the failing test. Let me try that out.

Reply:
Oh my bad I didn't read your lastq question properly. The test does fail if we remove this as the command sleeps for 3600 seconds.

Reply:
I have updated it with comments explaing how this test works.